### ISSUE TITLE
smbrkb5pwd is making bogus samba hashes.

### ISSUE THREAD

**Issue by jeffbearer** [opinsys/smbkrb5pwd#1]
I'm having a strange problem where i'm getting bogus hashes in my sambaNtPassword and sambaLmPassword fields.  I'm not able to authenticate with those hashes.   However if i put in a hash created by smbencrypt authentication against LDAP works as expected.

This link describes it exactly,  I have an email into the original poster to see if he came up with, and recalls his solution from 2009.

http://www.openldap.org/lists/openldap-technical/200910/msg00003.html

**Comment by Fraterius** [comment]
Yup the same problem in here with latest Debian, maybe it'd be nice to fix the Makefile. What I did is changed this line in Makefile:

SSL_LIB=-lcrypto -lssl
OPT=-g -DHAVE_OPENSSL -O2

Now Samba passwords are in sync correctly.
